script.js

```diff
@@ -3,59 +3,57 @@
     
     $("form").on("submit", function(event) {
         event.preventDefault();
+        var searchCity = $("#cityInput").val().trim();
 
         // Pull Wiki info
-        var searchArtist = $("#artistInput").val().trim();
-        var wikiURL = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch=" + searchArtist + "&srprop=snippet&format=json&origin=*"
-
-        $("#artistInfo").empty();
+        var wikiURL = "https://en.wikipedia.org/w/api.php?action=query&prop=extracts&exsentences=25&exlimit=3&titles=" + searchCity + "&explaintext=1&formatversion=2&format=json&origin=*"
+        
+        // "https://en.wikipedia.org/w/api.php?action=query&prop=extracts&exsentences=10&exlimit=1&titles=" + searchCity + "&explaintext=1&formatversion=2&format=json&origin=*"
+        
+        // "https://en.wikipedia.org/w/api.php?action=parse&page=" + searchCity + "&prop=wikitext&formatversion=2&format=json&origin=*" 
+        
+        // "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch=" + searchCity + "&srprop=snippet&format=json&origin=*"
+              
+        // "https://en.wikipedia.org/w/api.php?action=parse&page=" + searchCity + "&format=json&origin=*"
+        
```

Comment:
old wiki links, still need to be modified? will remove once we have perfected this formula